SwiftStaticLibrary/SwiftStaticLibrary.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SwiftStaticLibrary.swift
```

Comment:
Swift code in a static library with mixed code.  This is working to some extent, but I'm not sure how to access it from Objective-C in another module.  Inside the module, (Swift code accessing Objective-C) is working fine (due to bridging header).  The other direction works as well ([Swift header](https://github.com/calandis/SwiftStaticLibrarySample/pull/1/files#diff-1d02ffff23d3bf096be4f0f5b29a7decR9) import working as expected).